config-staging.json

```diff
@@ -89,6 +89,46 @@
       "supportedBy": [],
       "type": "EVM"
     },
+    {
+      "coinApi": "evm",
+      "explorerUrl": "https://arbiscan.io/tx",
+      "feeCurrencies": [
+        "native"
+      ],
+      "identifiers": {
+        "chainId": 42161
+      },
+      "memos": false,
+      "name": "Arbitrum",
+      "nativeAsset": "ETH.ARB",
```

Comment:
I think this should be `ARBETH`
https://github.com/blockchain/coin-definitions/blob/master/coins.json#L43